### ISSUE TITLE
Error: integer sizing security checks lead to invalid constraints

### ISSUE THREAD

**Issue by sleightsec** [42Crunch/vscode-openapi#85]
Given an integer parameter where type is set to "integer" and "format" is int64
```json
                    {
                        "name": "petId",
                        "in": "path",
                        "description": "ID of pet that needs to be updated",
                        "required": true,
                        "type": "integer",
                        "format": "int64",
                        "minimum": 0
                    },
```
The following audit entry is given:

Numeric parameter X of type 'integer' has no maximum defined.

Ok, great, we can define a maximum - in this case I want to make it = to the physical limits of a signed int64. New object:
```json
                    {
                        "name": "petId",
                        "in": "path",
                        "description": "ID of pet that needs to be updated",
                        "required": true,
                        "type": "integer",
                        "format": "int64",
                        "minimum": 0,
                        "maximum": 9223372036854775807
                    },
```
Now it appears that crashes 42crunch platform application. Submitting the updated json via VSCode plugin now returns only scores of 0 (does not analyze or display any errors other than sizing of the maximum value)
Audit error seen: "The value of the numeric property 'maximum' is too big for the type 'integer'

Expected outcome: Either
a) no audit error is created for a type that 42crunch platform cannot handle
b) crunch42 platform can handle signed int64 values

**Comment by ak1394** [comment]
Thanks for reporting! Currently we threat this value as an error, which stops further evaluation of the OpenAPI file. We're looking into this issue.

**Comment by ak1394** [comment]
The problem is caused by the extension itself, which (to implement the support for multi-file OpenAPI files, etc) converts OpenAPI file before sending it for the audit.

During the conversion, the "maximum" value of 9223372036854775807 (which is bigger than [MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) gets converted to a bigger number which causes the audit error.

For the time being I suggest lowering the "maximum" value to 9007199254740991 (MAX_SAFE_INTEGER) which would also guarantee your API compatibility with Javascript clients (browsers/node,etc)

**Comment by ak1394** [comment]
Fixed in 4.6.1
